src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js

```diff
@@ -587,6 +587,18 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
             $this.checkMatchedItem = false;
         });
 
+        this.panel.on('click', function() {
```

Comment:
make it `click.emptyMessage` I like to add namespaces so when debugging we can easily tell where the onClick came from